<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageRotatePara</class>
 <widget class="QWidget" name="imageRotatePara">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>265</height>
     </rect>
    </property>
    <attribute name="label">
     <string>图像选取</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>281</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>保存配置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reductionButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/main.qrc">
            <normaloff>:/image/reback.svg</normaloff>:/image/reback.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="ImgSelectButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/main.qrc">
            <normaloff>:/image/juxing.svg</normaloff>:/image/juxing.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mouseButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../res/main.qrc">
            <normaloff>:/image/shubiaojiantou_1.svg</normaloff>:/image/shubiaojiantou_1.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="binarization" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>265</height>
     </rect>
    </property>
    <attribute name="label">
     <string>二值化</string>
    </attribute>
   </widget>
   <widget class="lineFeature" name="page_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>265</height>
     </rect>
    </property>
    <attribute name="label">
     <string>直线提取</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="page_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>265</height>
     </rect>
    </property>
    <attribute name="label">
     <string>比对</string>
    </attribute>
    <widget class="QPushButton" name="MatchButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>135</y>
       <width>93</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>比对</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20px &quot;微软雅黑&quot;;
color:#444444;</string>
     </property>
     <property name="text">
      <string>参考图:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20px &quot;微软雅黑&quot;;
color:#444444;</string>
     </property>
     <property name="text">
      <string>结果计算：</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ImgSelectButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/main.qrc">
          <normaloff>:/image/juxing.svg</normaloff>:/image/juxing.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MouseButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/main.qrc">
          <normaloff>:/image/shubiaojiantou_1.svg</normaloff>:/image/shubiaojiantou_1.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>binarization</class>
   <extends>QWidget</extends>
   <header location="global">view/binarization.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lineFeature</class>
   <extends>QWidget</extends>
   <header>view/linefeature.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/main.qrc"/>
 </resources>
 <connections/>
</ui>
